Tests/Unit/FirstClassTest.php

```diff
@@ -4,6 +4,9 @@
 use Helhum\ExtScaffold\Tests\Unit\Fixtures\LoadableClass;
 use Nimut\TestingFramework\TestCase\UnitTestCase;
 
+/**
+ * Test case.
```

Comment:
that comment does not provide any value, I'd rather skip it

Reply:
Done.